pkgs/by-name/ld/ld64/patches/0010-Provide-mach-compatibility-headers-based-on-LLVM-s-h.patch

```diff
@@ -0,0 +1,662 @@
+From 74e5584dac17679413dcfee76d4df1943cca566b Mon Sep 17 00:00:00 2001
+From: Randy Eckenrode <randy@largeandhighquality.com>
+Date: Wed, 13 Nov 2024 13:53:14 -0500
+Subject: [PATCH 10/20] =?UTF-8?q?Provide=20mach=20compatibility=20headers?=
```

Comment:
Should the `src/abstraction/MachOFileAbstraction.hpp` change be part of this commit? And would it maybe be possible/better to use some globs with `using` to cut down on the size of these? (Not a big deal though.)

Reply:
You can’t mix LLVM headers and Darwin mach-o headers. LLVM does something similar internally. I tried using broader `using` statements, but that brought in too much stuff and broke the build in other ways.